spyder/plugins/appearance/confpage.py

```diff
@@ -321,27 +319,32 @@ def update_preview(self, index=None, scheme_name=None):
         """
         text = ('"""A string"""\n\n'
                 '# A comment\n\n'
-                '# %% A cell\n\n'
                 'class Foo(object):\n'
                 '    def __init__(self):\n'
                 '        bar = 42\n'
                 '        print(bar)\n'
                 )
-        show_blanks = CONF.get('editor', 'blank_spaces')
-        update_scrollbar = CONF.get('editor', 'scroll_past_end')
-        underline_errors = CONF.get('editor', 'underline_errors')
+
         if scheme_name is None:
             scheme_name = self.current_scheme
-        self.preview_editor.setup_editor(linenumbers=True,
-                                         markers=True,
-                                         tab_mode=False,
-                                         font=get_font(),
-                                         show_blanks=show_blanks,
-                                         underline_errors=underline_errors,
-                                         color_scheme=scheme_name,
-                                         scroll_past_end=update_scrollbar)
-        self.preview_editor.set_text(text)
+
+        # FIXME: The appearance plugin is not tied to the code editor
+        # so these calls here probably should not be used, since it
+        # assumes there is an editor, which might not be the case at
+        # all. We can probably make a decission for the user and
+        # that should be good enough, after all this is just a quick
+        # preview.
+        show_blanks = CONF.get('editor', 'blank_spaces')
+        update_scrollbar = CONF.get('editor', 'scroll_past_end')
```

Comment:
Let's take those decisions now: please make `show_blanks` and `update_scrollbar` `False` in the preview.

Reply:
👍

Reply:
Fixed.